src/css3/new_tokenizer.rs

```diff
@@ -0,0 +1,161 @@
+// note: input_stream should come from a shared lib.
+use crate::html5_parser::input_stream::InputStream;
```

Comment:
I agree with the comment.  Although I can also see there being an html5 token stream and a css3 token stream, and perhaps they implement a shared interface (don't have any insights at this point).